main/iio-sensor-proxy/template.py

```diff
@@ -0,0 +1,45 @@
+pkgname = "iio-sensor-proxy"
+pkgver = "3.5"
+pkgrel = 0
+build_style = "meson"
+configure_args = [
+    "-Dgeoclue-user=_geoclue",
+    "-Dsystemdsystemunitdir=",
+    "-Dtests=true",
+]
+hostmakedepends = [
+    "meson",
+    "pkgconf",
+]
+makedepends = [
+    "glib-devel",
+    "libgudev-devel",
+    "linux-headers",
+    "polkit-devel",
+    "udev-devel",
+]
+checkdepends = [
+    "python-dbusmock",
+    "python-gobject",
+    "python-psutil",
+    "umockdev",
+]
+install_if = [f"iio-sensor-proxy-meta={pkgver}-r{pkgrel}"]
+pkgdesc = "IIO accelerometer sensor to input device proxy"
+maintainer = "Jami Kettunen <jami.kettunen@protonmail.com>"
+license = "GPL-3.0-or-later"
+url = "https://gitlab.freedesktop.org/hadess/iio-sensor-proxy"
+source = f"{url}/-/archive/{pkgver}/iio-sensor-proxy-{pkgver}.tar.gz"
+sha256 = "8689425f2287626a95d95b1e1e5b62e497d09dd08cf411084ed22166d4a49da5"
+hardening = ["vis", "cfi"]
+
+
+def post_install(self):
+    self.install_service(self.files_path / "iio-sensor-proxy", enable=True)
```

Comment:
should it be enable=true?

Reply:
idk, ideally it'd be dbus activated similar to https://gitlab.freedesktop.org/hadess/iio-sensor-proxy/-/blob/master/data/iio-sensor-proxy.service.in, ig until we have that with `dbus-broker` and the like sensor functionality should need `doas dinitctl enable iio-sensor-proxy`

Reply:
you can do that by adding a dbus service file (since they don't ship one) and putting dinitctl start in it (see some other stuff that does that)

Reply:
i would guess one isn't shipped because the udev rules file adds a systemd wants for the service, the dbus= systemd type is just for readiness notification

(and that has the effect of only starting it if sensors were detected on systemd)

best we can do is just add a normal dbus service that calls dinitctl start when something looks up the name and hence autostarts it

Reply:
made it a triggered service that's enabled by default (on desktops it'll show up as `STARTING` and makes `doas dinitctl list` look meh) where the udev rule does `dinitctl trigger iio-sensor-proxy` to actually launch it when sensors are detected similar to systemd.

alternatively it could just be a default-disabled service and have the udev rule `dinitctl start iio-sensor-proxy` on detected sensors but not sure which is more preferred